src/agent/debuglet.ts

```diff
@@ -514,32 +512,16 @@ export class Debuglet extends EventEmitter {
     return project;
   }
 
-  static async runningOnGCP(): Promise<boolean> {
-    const lookup = promisify(dns.lookup);
```

Comment:
Neat!  Do you think this is a better approach than a GET?  If so, I can submit a PR to use this method.

Reply:
I don't think it is worth changing. DNS Lookup is faster; but it doesn't care about whether the metadata service is actually a 'correct' metadata service; i.e. it doesn't care about the headers.